test/suite/helper_tiered.py

```diff
@@ -55,27 +55,42 @@ def get_auth_token(storage_source):
 # Local buckets do not have a region
 local_buckets = ['bucket1', 'bucket2']
 
-# Get name of the first bucket in the list.
-def get_bucket1_name(storage_source):
+# Get name of the bucket at specified index in the list.
+def get_bucket_name(storage_source, i):
     if storage_source == 's3_store':
-        return s3_buckets[0]
+        return s3_buckets[i]
     if storage_source == 'dir_store':
-        return local_buckets[0]
+        return local_buckets[i]
     return None
 
-# Get name of the second bucket in the list.
-def get_bucket2_name(storage_source):
-    if storage_source == 's3_store':
-        return s3_buckets[1]
-    if storage_source == 'dir_store':
-        return local_buckets[1]
-    return None
+# Set up configuration
+def get_conn_config(storage_source):
+    if not storage_source.is_tiered_scenario():
+            return ''
+    if storage_source.ss_name == 'dir_store' and not os.path.exists(storage_source.bucket):
+            os.mkdir(storage_source.bucket)
+    return \
+        'debug_mode=(flush_checkpoint=true),' + \
+        'statistics=(all),' + \
+        'tiered_storage=(auth_token=%s,' % storage_source.auth_token + \
+        'bucket=%s,' % storage_source.bucket + \
+        'bucket_prefix=%s,' % storage_source.bucket_prefix + \
+        'name=%s,' % storage_source.ss_name
+
+def get_check(storage_source, tc, base, n):
+    for i in range(base, n):
+        storage_source.assertEqual(tc[str(i)], str(i))
+    tc.set_key(str(n))
+    storage_source.assertEquals(tc.search(), wiredtiger.WT_NOTFOUND)
 
 # Generate a unique object prefix for the S3 store. 
 def generate_s3_prefix(test_name = ''):
     # Generates a unique prefix to be used with the object keys, eg:
-    # "s3test/2022-31-01-16-34-10_623843294/"
-    prefix = 's3test/'
+    # "s3test_artefacts--python_2022-31-01-16-34-10_623843294/".
```

Comment:
You can update the pattern example on line 89

Reply:
Hi Sulabh, thanks for leaving your review. I have updated the pattern example.